core/src/main/scala/org/apache/spark/deploy/client/TestClient.scala

```diff
@@ -49,7 +49,8 @@ private[spark] object TestClient {
     val conf = new SparkConf
     val rpcEnv = RpcEnv.create("spark", Utils.localHostName(), 0, conf, new SecurityManager(conf))
     val desc = new ApplicationDescription("TestClient", Some(1), 512,
-      Command("spark.deploy.client.TestExecutor", Seq(), Map(), Seq(), Seq(), Seq()), "ignored")
+      Command("org.apache.spark.deploy.client.TestExecutor", Seq(), Map(), Seq(), Seq(), Seq()),
```

Comment:
Perhaps a naive question, but why not use classOf[TextExecutor].getCanonicalName so that this doesn't break in the future and is easier to grep with IntelliJ?